src/SyliusPayPalPlugin.php

```diff
@@ -24,4 +25,10 @@ public function getPath(): string
     {
         return \dirname(__DIR__);
     }
+
+    // This method is overridden to change the alias to "sylius_paypal"
+    public function getContainerExtension(): ?ExtensionInterface
```

Comment:
```suggestion
    // This method is overridden to change the alias to "sylius_paypal"
    // instead of the autogenerated "sylius_pay_pal"
    public function getContainerExtension(): ?ExtensionInterface
```